vnext/ReactUWP/Views/KeyboardEventHandler.cpp

```diff
@@ -138,7 +138,7 @@ HandledKeyboardEventHandler::HandledKeyboardEventHandler()
 {
 }
 
-void HandledKeyboardEventHandler::UpdateHandledKeyboardEvents(string const& propertyName, folly::dynamic const& value)
+void HandledKeyboardEventHandler::UpdateHandledKeyboardEvents(std::string const& propertyName, folly::dynamic const& value)
```

Comment:
Feel free to add a using for std in cpp. That's fine. It just shouldn't be in the header.